service/src/test/java/uk/gov/hmcts/reform/fpl/service/MigrateCaseServiceTest.java

```diff
@@ -748,4 +753,80 @@ void shouldThrowExceptionWithoutCourt() {
                 .hasMessage("The case does not have court or local authority policy's organisation.");
         }
     }
+
+
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    @Nested
+    class RevertChildExtensionDate {
+        LocalDate completeDate = LocalDate.of(2023, 1, 1);
+        LocalDate revertedDate = LocalDate.of(2022, 1, 1);
+
+        CaseExtensionReasonList extensionReason = CaseExtensionReasonList.DELAY_IN_CASE_OR_IMPACT_ON_CHILD;
+        CaseExtensionReasonList revertedReason = CaseExtensionReasonList.NO_EXTENSION;
```

Comment:
Maybe add a case for setting null to revertedReason?

Reply:
done